src/test/key_tests.cpp

```diff
@@ -352,7 +356,7 @@ BOOST_AUTO_TEST_CASE(key_ellswift)
 
 BOOST_AUTO_TEST_CASE(bip341_test_h)
 {
-    std::vector<unsigned char> G_uncompressed = ParseHex("0479be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798483ada7726a3c4655da4fbfc0e1108a8fd17b448a68554199c47d08ffb10d4b8");
+    constexpr auto G_uncompressed{"0479be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798483ada7726a3c4655da4fbfc0e1108a8fd17b448a68554199c47d08ffb10d4b8"_hex};
     HashWriter hw;
     hw.write(MakeByteSpan(G_uncompressed));
```

Comment:
nit in 6954b1f1143a73d4024f5564e287d8eeb34ff577: no more need for `MakeByteSpan`:

```suggestion
    hw.write(G_uncompressed);
```

Reply:
Done!